task3_10.py

```diff
@@ -0,0 +1,19 @@
+
+a = list(map(int, input('Enter your numbers through spaces: ').split()))
+lst = []
+i = 1
+for i in a:
+    if i % 2 ==0:
+        continue
+    if i % 3 == 0:
+        continue
+    if i % 5 == 0:
+        continue
+    if i % 7 == 0:
+        continue
+    if i % 9 == 0:
+        continue
+    else:
+        lst.append(i)

```

Comment:
what if there's a number greater than 9? it should be covered by an additional nested cycle, I suggest you to look through solutions of your colleagues